Build.PL

```diff
@@ -60,13 +60,16 @@ my $builder = MyBuild->new(
         'Hash::FieldHash'     => '0.06',
         'Path::Tiny'          => '0.036',
         'Import::Into'        => '1.002003',
+        'Sub::Name'           => '0',
+        'Time::y2038'         => '0',        
```

Comment:
Time::y2038 is a conditional dependency.

    $builder->requires->{"Time::y2038"} = "20100218" if $builder->needs_y2038;

That might be an unnecessary complication, but that should be examined in its own issue.